cmd/tikv-ctl/src/main.rs

```diff
@@ -2035,67 +1823,78 @@ fn main() {
             toml::from_str(&s).unwrap()
         },
     );
-    let mgr = new_security_mgr(&matches);
-
-    // Bypass the ldb command to RocksDB.
-    if let Some(cmd) = matches.subcommand_matches("ldb") {
-        run_ldb_command(cmd, &cfg);
-        return;
-    }
+    let mgr = new_security_mgr(&opt);
+
+    let cmd = match opt.cmd {
+        Some(cmd) => cmd,
+        None => {
+            // Deal with arguments about key utils.
```

Comment:
It seems this part is duplicated with L1873?

Reply:
Yes they are duplicate code. This is due to a merge mistake. Fixed.